tests/e2e/framework/testInfo.go

```diff
@@ -35,6 +36,7 @@ import (
 
 var (
 	logsBucketPath = flag.String("logs_bucket_path", "", "Cloud Storage Bucket path to use to store logs")
+	testLogsPath   = flag.String("test_log_path", "", "Local path to store logs in")
```

Comment:
ahh i see the original typo came from that the var is testLogsPath whereas flag is test_log_path